Skip filters for health, token and other similar APIs (#165)
wso2/docs-mg: en/docs/how-tos/extensions/custom-filters.md

# Add Custom Filters

> Filters are set of execution points in the request and response flow which will intercept the request before going to the backend service and intercept the response before forwarding to the client.

Reviewer: ```suggestion
Filters are set of execution points in the request and response flow that intercept the request before it goes to the backend service and intercepts the response before forwarding to the client.
```

    </tbody>
</table> 

4.Then build the project. Custom filter will be engaged in the runtime.

##  Skip endpoints for custom filters

To skip filters for health, token, and other similar APIs you can use the filter implementation given below.

1.  Add the following inside the `filterRequest` and `filterResponse` methods.

```
string SKIP_ALL_FILTERS = "skip_filters";
    if (context.attributes.hasKey(SKIP_ALL_FILTERS) && <boolean>context.attributes[SKIP_ALL_FILTERS]) {
    return true;
    }

> ```

Reviewer: Let's tab the code blocks.

> 2.  Go to the `deployment.toml` file and add the configuration given below, to set the filter position parameter.

Reviewer: ```suggestion
2.  To set the filter position parameter, go to the `deployment.toml` file and add the configuration given below.
```
